popup/js/model/options.js

```diff
@@ -279,6 +287,10 @@ export const defaultOptions = {
    * This is set very high to ensure that it's the topmost entry
    */
   scoreCustomSearchEngineBaseScore: 500,
+  /**
+   * Base score for a direct URL being typed in
+   */
+  scoreDirectUrlScore: 1000,
```

Comment:
```suggestion
  scoreDirectUrlScore: 500,
```

I think 500 should be enough, then also the overall number shouldn't exceed 4 chars. But agree that it should always be first result.